client/src/components/payment/fee-summary.vue

```diff
@@ -1,70 +1,74 @@
 <template>
-  <v-card>
-    <header class="font-weight-bold px-3 py-3">
-      <slot name="header">Payment Details</slot>
-    </header>
-
-    <v-alert v-if="fetchError" color="error" icon="mdi-alert" outlined class="my-0" v-html="fetchError" />
-
-    <v-slide-y-transition group tag="ul" class="fee-list" v-show="!fetchError">
-      <template
-        v-for="lineItem in fees"
-        v-show="(totalFees > 0 && lineItem.filingFees) || (totalFees === 0)"
-        >
-        <li class="container fee-list__item"
-          :key="lineItem.filingType"
-          >
-          <div class="fee-list__item-name">{{lineItem.filingType}}</div>
-          <div class="fee-list__item-value" v-if="lineItem.filingFees > 0">${{lineItem.filingFees.toFixed(2)}}</div>
-          <div class="fee-list__item-value" v-else>No Fee</div>
-        </li>
-        <li class="container fee-list__item"
-          v-if="lineItem.priorityFees"
-          :key="lineItem.filingTypeCode+'-priority'"
-          >
-          <div class="fee-list__item-name pl-3">Priority Fee</div>
-          <div class="fee-list__item-value">${{lineItem.priorityFees.toFixed(2)}}</div>
-        </li>
-        <li class="container fee-list__item"
-          v-if="lineItem.serviceFees"
-          :key="lineItem.filingTypeCode+'-transaction'"
-          >
-          <div class="fee-list__item-name pl-3">Service Fee</div>
-          <div class="fee-list__item-value">${{lineItem.serviceFees.toFixed(2)}}</div>
-        </li>
-      </template>
-    </v-slide-y-transition>
-
-    <div class="container fee-total" v-show="!fetchError">
-      <div class="fee-total__name">Fees</div>
-      <!--<div class="fee-total__currency">CAD</div>-->
-      <div class="fee-total__value">
-        <v-slide-y-reverse-transition name="slide" mode="out-in">
-          <div>${{totalFees.toFixed(2)}}</div>
-        </v-slide-y-reverse-transition>
-      </div>
-    </div>
-
-    <div class="container fee-total tax-total" v-show="!fetchError">
-      <div class="fee-total__name">Tax</div>
-      <!--<div class="fee-total__currency">CAD</div>-->
-      <div class="fee-total__value">
-        <v-slide-y-reverse-transition name="slide" mode="out-in">
-          <div>${{totalTax.toFixed(2)}}</div>
-        </v-slide-y-reverse-transition>
-      </div>
-    </div>
-
-    <div class="container fee-total payment-total" v-show="!fetchError">
-      <div class="fee-total__name">Total</div>
-      <!--<div class="fee-total__currency">CAD</div>-->
-      <div class="fee-total__value">
-        <v-slide-y-reverse-transition name="slide" mode="out-in">
-          <div><b>${{total.toFixed(2)}}</b></div>
-        </v-slide-y-reverse-transition>
-      </div>
-    </div>
-  </v-card>
+  <div class="fee-summary">
+    <v-row align="center" justify="center">
+      <v-col cols="10">
+        <div class="font-weight-bold px-3 py-3 header">
+          Payment Details
+        </div>
```

Comment:
If possible, it's better to have the tags and text on the same line.

The reason is, we can test for "toBe()" with exact text. Otherwise we have to test with "toContain()" since there may be newlines in the text content.

(Similarly, you will sometimes see text within span tags that don't seem to do anything, but they help with testing.)